src/pages/groupComparison/GroupComparisonMutationTable.tsx

```diff
@@ -0,0 +1,310 @@
+import * as React from 'react';
+import {
+    IMutationTableProps,
+    MutationTableColumnType,
+    default as MutationTable,
+    MutationTableColumn,
+} from 'shared/components/mutationTable/MutationTable';
+import { createMutationNamespaceColumns } from 'shared/components/mutationTable/MutationTableUtils';
+import _ from 'lodash';
+import { adjustVisibility } from 'shared/components/alterationsTableUtils';
+import { getServerConfig } from 'config/config';
+import {
+    ComparisonGroup,
+    SIGNIFICANT_QVALUE_THRESHOLD,
+} from './GroupComparisonUtils';
+import ComplexKeyMap from 'shared/lib/complexKeyDataStructures/ComplexKeyMap';
+import { Mutation, Sample } from 'cbioportal-ts-api-client';
+import GroupMutatedCountPercentageColumnFormatter from 'shared/components/mutationTable/column/GroupMutatedCountPercentageColumnFormatter.tsx';
+import LogRatioColumnFormatter from 'shared/components/mutationTable/column/LogRatioColumnFormatter';
+import EnrichedInColumnFormatter from 'shared/components/mutationTable/column/EnrichedInColumnFormatter';
+import AlterationOverlapColumnFormatter from 'shared/components/mutationTable/column/AlterationOverlapColumnFormatter';
+import PValueColumnFormatter from 'shared/components/mutationTable/column/PValueColumnFormatter';
+import { ComparisonMutationsRow } from 'shared/model/ComparisonMutationsRow';
+import QValueColumnFormatter from 'shared/components/mutationTable/column/QValueColumnFormatter';
+
+export interface IGroupComparisonMutationTableProps
+    extends IMutationTableProps {
+    // add comparison view specific props here if needed
+    isCanonicalTranscript: boolean | undefined;
+    profiledPatientCounts: number[];
+    groups: ComparisonGroup[];
+    sampleSet: ComplexKeyMap<Sample>;
+    rowDataByProteinChange: {
+        [proteinChange: string]: ComparisonMutationsRow;
+    };
+}
+
+export default class GroupComparisonMutationTable extends MutationTable<
+    IGroupComparisonMutationTableProps
+> {
+    public static defaultProps = {
+        ...MutationTable.defaultProps,
+        columns: [
+            MutationTableColumnType.ANNOTATION,
+            MutationTableColumnType.MUTATION_STATUS,
+            MutationTableColumnType.PROTEIN_CHANGE,
+            MutationTableColumnType.MUTATION_TYPE,
+            '(A) Group',
+            '(B) Group',
+            'Log2 Ratio',
+            'Enriched in',
+            'Alteration Overlap',
+            'P_VALUE',
+            'Q_VALUE',
```

Comment:
We should probably define these extra column names as constants/enums somewhere as well.